src/app/features/tasks/task/task.component.scss

```diff
@@ -116,18 +116,11 @@
     }
   }
 
-  @include isHostFocus('.inner-wrapper') {
+  :host:focus > .inner-wrapper > & {
     border-radius: $task-border-radius;
-    // because of the border, this looks better
-    //top: 2px;
-    //bottom: 2px;
-
-    @include mq(xs) {
-      // NOTE: somehow ng build messes up, if we don't include it like that
-      border-color: $task-c-focus;
-      border-width: 1px;
-      border-style: solid !important;
-    }
```

Comment:
Not perfectly sure if this is OK, considering your comment about ng build messing up... I didn't notice any issues and it solved my problem of items not showing focus style on mobile...

Reply:
I think we need to test this for the actual electron build (the one created by `yarn buildAllElectron:prod && electron-builder`). I remember this not working there though this might be not a problem anymore as electron's chromium version has changed several times since.